wolox-books/src/app/screens/auth/components/books-service/books.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { LocalStorageService } from '../../../../components/local-storage/local-storage.service';
+import { environment } from '../../../../../environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BooksService {
+
+  constructor(private http: HttpClient, private localStorage: LocalStorageService) { }
+
+  public getBooks(id = ""): Observable<Response> {
```

Comment:
Is it really necessary to pass the `id` as blank to get the books list? 🤔

Reply:
I think it's not really self explanatory to use the same method to list the books as the one that gets the book details